packages/tools/client-debugger/client-debugger/src/Registry.ts

```diff
@@ -122,6 +125,41 @@ export class DebuggerRegistry extends TypedEventEmitter<DebuggerRegistryEvents>
 			this.postRegistryChange();
 			return true;
 		},
+		["CONNECT_CONTAINER"]: (untypedMessage) => {
```

Comment:
I don't think we want to handle these events on the Registry. I would recommend adding these handlers to the FluidClientDebugger itself. When it receives an incoming message, it can ignore it if the message is for a different container ID. FluidClientDebugger should already have an example of this for getting basic stat information (the `GET_CONTAINER_STATE` message).

Reply:
Once moved, we should also add documentation to FluidClientDebugger about the newly supported messages. Its documentation should already contain examples for its existing messages.

Reply:
Got it. Updated accordingly. Thx!